### PR TITLE
Chore: dynamic crypto providers

### PR DESCRIPTION
**Pull request by brendan-defi**
**What changed? Why?**
* Created client-side-loading DynamicCryptoProviders to remove crypto JS bloat from certain pages

**Notes to reviewers**

**How has it been tested?**
locally

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/app/CryptoProviders.dynamic.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { useErrors } from 'apps/web/contexts/Errors';
+
+// RigidBody cannot be imported using dynamic() due to some import issues
```
Comment (zencephalon) [inline apps/web/app/CryptoProviders.dynamic.tsx:6]: But

Comment (zencephalon) [reply apps/web/app/CryptoProviders.dynamic.tsx:6]: Nit

Comment (brendan-defi) [reply apps/web/app/CryptoProviders.dynamic.tsx:6]: thanks, removing

**Comment by brendan-defi** [comment]
closing in favor of new version that solves for style issues: https://github.com/base-org/web/pull/1248

**Pull request closed without merging**